core/iwasm/libraries/libc-wasi/libc_wasi_wrapper.c

```diff
@@ -192,7 +192,7 @@ wasi_clock_res_get(wasm_exec_env_t exec_env,
     if (!validate_native_addr(resolution, sizeof(wasi_timestamp_t)))
         return (wasi_errno_t)-1;
 
-    return wasmtime_ssp_clock_res_get(clock_id, resolution);
+    return os_clock_res_get(clock_id, resolution);
```

Comment:
I think this should stay as it is, i.e.
```suggestion
    return wasmtime_ssp_clock_res_get(clock_id, resolution);
```